src/main/java/com/gmail/goosius/siegewar/listeners/SiegeWarSelfListener.java

```diff
@@ -18,4 +23,93 @@ public void onBattleSessionPreStart(BattleSessionPreStartEvent event) {
 			event.setCancellationMsg(Translatable.of("battle_session_cancelled_no_sieges").defaultLocale());
 		}
 	}
+
+	@EventHandler
+	public void onBattleSessionStarted(BattleSessionStartedEvent event) {
+		if (!SiegeWarSettings.isDiscordWebhookEnabled())
+			return;
+
+		DiscordWebhook webhook = new DiscordWebhook(SiegeWarSettings.getDiscordWebhookUrl());
+
+		webhook.addEmbed(new DiscordWebhook.EmbedObject()
+				.setColor(new Color(255, 157, 0))
+				.setDescription(event.getMessage().substring(2))

```

Comment:
Why does this and the Ended method both use substring(2)?

Reply:
because then it has the weird s thing for the color like `§b`